src/sonic-yang-models/tests/yang_model_tests/yangTest.json

```diff
@@ -1128,6 +1128,9 @@
                 },
                 "DEBUG_COUNTER": {
                     "FLEX_COUNTER_STATUS": "enable"
+                },
```

Comment:
I think, the file uses tabs, not spaces, that`s why the alignment mismatches. Even the previous field is already misaligned. If you can fix it it will be great. Otherwise the diff looks good to me.

Reply:
I replace space with tab in all file.